stl/inc/array

```diff
@@ -617,8 +617,10 @@ public:
     _DEPRECATE_TR1_NAMESPACE void assign(const _Ty&) {}
 #endif // _HAS_TR1_NAMESPACE
 
+    template <class _OTy = _Ty, enable_if_t<!is_const_v<_OTy>, int> = 0>
     _CONSTEXPR20 void fill(const _Ty&) {}
 
+    template <class _OTy = _Ty, enable_if_t<!is_const_v<_OTy>, int> = 0>

```

Comment:
1. Ditto member `swap` typically doesn't SFINAE since `is_swappable` and the `swappable` concept don't look at member  `swap`.

2. Ditto `is_const_v` is wrong, I think we want to `static_assert` `_Is_swappable_v<_Ty>` (which is equivalent to C++17 `is_swappable_v`, but available in all language modes).

Reply:
@CaseyCarter I can't find `_Is_swappable_v<_Ty>`. Could you please let me know its header file? Thank you in advance.

Reply:
(Peeks at `<type_traits>`) My mistake! We don't have a `_v` variable template, only the `_Is_swappable` class template flavor, so you'll need to use `_Is_swappable<_Ty>::value`.